### PR TITLE
Blockly rulecontext types

### PR DESCRIPTION
**Pull request by stefan-hoehn**
As discussed in https://community.openhab.org/t/blocky-in-text-block-rejects-contextual-info-triggering-item-name-in-4-0-worked-in-3-4-4/147980/10  a more clear description of what is returned with rule context information block.

### CODE REVIEW

File: configuration/blockly/rules-blockly-run-and-process.md [hunk 1]
```diff
@@ -81,9 +81,22 @@ _Function:_ Retrieve event context related information of the rule
 
 A rule carries contextual information when triggered - this block can retrieve that information.
 
-- The rule trigger type defines which of the attributes are available to the rule - they will not all be available.
+- The rule trigger type defines which of the attributes are available to the rule - not not all be available.
```
Comment (jlaur) [inline configuration/blockly/rules-blockly-run-and-process.md:84]: Not sure what was intended here, but it seems something went wrong ("not not all").

**Comment by Confectrician** [comment]
> @Confectrician Even though I synced the fork and pulled main before I created the branch I seem to have 8 commits with this one. Do you like me to close this one and create a new PR instead?

No it's fine.

File: configuration/blockly/rules-blockly-run-and-process.md [hunk 2]
```diff
@@ -81,9 +81,25 @@ _Function:_ Retrieve event context related information of the rule
...
+- triggering item name - [Item](rules-blockly-items-things.html#item), not a String
+- received command - String / Number by selection
+- triggered channel - String
+- triggered event - String
+
+The block adds a selection drop down in case a state or command is returned:
+
+![blockly-context-number-string](../images/blockly/blockly-context-number-string.gif)
+
+
```
Comment (Confectrician) [inline configuration/blockly/rules-blockly-run-and-process.md:102]:
```suggestion
```

**Review by Confectrician — COMMENTED** [review]
PLease remove this line to fix the markdown linting error.

**Review by Confectrician — APPROVED** [review]
Thanks. 👍🏼

**Pull request merged**